pkgs/build-support/rust/hooks/default.nix

```diff
@@ -70,13 +70,17 @@ in {
 
         # Target platform
         rustTarget = ''
```

Comment:
Might make sense to rename this, and probably expand the comment a bit to explain what's going on?  You could probably just adapt what I wrote in the other PR if you want.

Reply:
done, please let me know if the comment is ok